ChangeLog

```diff
@@ -1,3 +1,93 @@
+TBD Version 4.7.0 '':
+Contributors:
+	- Arjun (36335769+0x34d@users.noreply.github.com)
+	- Emil Ljungdahl (111423223+pando-emil@users.noreply.github.com)
+	- Gregor Jasny (gjasny@googlemail.com)
+	- Gustavo Garcia (gustavogb@gmail.com)
+	- Hyorin Choi (17173216+hyorin@users.noreply.github.com)
+	- Joachim Bauch (bauch@struktur.de)
+	- Kang Lin (kl222@126.com)
+	- Paul Kramer (47924093+paulkram@users.noreply.github.com)
+	- Pavel Punsky (eakraly@users.noreply.github.com)
+	- Robert Scheck (robert-scheck@users.noreply.github.com)
+	- Samuel Marks (807580+SamuelMarks@users.noreply.github.com)
+	- Scott Godin (sgodin@sipspectrum.com)
+	- Thibaut ACKERMANN (thib-ack@users.noreply.github.com)
+	- Tom Bevan (thehorrorthehorror@gmail.com)
+	- Yoshiki Kadoshita (sublimer.me@gmail.com)
+	- tyranron (tyranron@gmail.com)
+
+Changelist:
+	- Do not build debug version during PR preflight (Pavel Punsky)
+	- Bump actions/cache from 2 to 3 (#1084) (dependabot[bot])
```

Comment:
@eakraly I think we may remove such lines from changelog after auto-generating as they make not much sense for readers and only pollute the list of changes. Changelog, imho, diverges from git log in the way that it should contain only changes visible for the end-users.

Reply:
True
I wanted to delete it but decided to keep it for review just in case